### PR TITLE
Sponsor advertising banner

### PR DESCRIPTION
**Pull request by zephyrich**
- add sponsor advertising banner to home page
- update sponsor contact info

### CODE REVIEW

File: _layouts/sponsorship.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
 <div class="hero px-0 py-10 p-md-8 p-lg-10 sponsorship-image">
     <div class="hero-content col-lg-6 col-10">
         <h1>诚邀招商：</h1>
-        <p>你是不是觉得，2020 - 2022年是隐没的两年？作为商家，你是不是苦于没有曝光，流量不足？酒香也怕巷子深。在此，“载歌在谷”携各位热情的志愿者小伙伴们，郑重推出2023年线下兔年春晚及游园活动！正在进行时，诚邀您的参加！</p>
+        <p>你是不是觉得，2020 - 2022年是隐没的两年？作为商家，你是不是苦于没有曝光，流量不足？酒香也怕巷子深。在此，“载歌在谷”携各位热情的志愿者小伙伴们，郑重推出2023载歌在谷夏季歌手赛及中秋游园会活动！正在进行时，诚邀您的参加！</p>
         <div id="hero-divider"></div>
-        <p>联系我们：sponsor-gala@zgzg.io</p>
+        <p>联系我们：<a href="http://zgzg.link/sponsor-2023-summer-form">http://zgzg.link/sponsor-2023-summer-form</a></p>
```
Comment (tslmy) [inline _layouts/sponsorship.html]:
```suggestion
        <p>联系我们：<a href="https://forms.gle/obCKZCDSDVx1TWCQ6">表单</a></p>
```

File: _sass/components/_intro.scss [hunk 2]
```diff
@@ -20,12 +20,13 @@
...
     font-size: 16px;
     font-weight: 400;
-    letter-spacing: 0.01em;
+    letter-spacing: .01em;
     line-height: 26px;
   }
 
   #content {
     background: $primary;
+    opacity: .9
```
Comment (tslmy) [inline _sass/components/_intro.scss:29]: Okay with me, but... we actually had designers intentionally designed this to be opaque. 

https://www.figma.com/file/7uczsKDoFUxT5nzLrwC9aI/2023-ZGZG%E6%98%A5%E6%99%9A%E7%BD%91%E7%AB%99%E7%BB%84?node-id=1%3A2

File: _includes/sponsor-ad-banner.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div class="row alert alert-success text-center mb-0 rounded-0" role="alert">
+    <h4>2023载歌在谷夏季歌手赛及中秋游园会火热招商中 <a href="/sponsorship/" class="alert-link">查看详情</a></h4>
```
Comment (tslmy) [inline _includes/sponsor-ad-banner.html]: nit: The message body and the link are kinda close. Punctuation or full-width space might make it look easier.

```suggestion
    <h4>2023 「载歌在谷」夏季歌手赛、中秋游园会火热招商中！<a href="/sponsorship/" class="alert-link">查看详情</a></h4>
```

Smart usage of the `alert-success` class!

I think the text could be better centered vertically... wdyt?

A final nit is `h4`. There's no `h1`, `h2`, or `h3` above it. It's not semantically sound, to my eyes. Is there a better way to define the text size?

**Review by tslmy — CHANGES_REQUESTED** [review]
commented

File: _sass/components/_intro.scss (hunk 2 above)
Comment (zephyrich) [reply _sass/components/_intro.scss:29]: Requested access. It seems we have no designers this year yet. 
Revised according to Lizzi and Elise's suggestions.

File: _includes/sponsor-ad-banner.html (hunk 3 above)
Comment (zephyrich) [reply _includes/sponsor-ad-banner.html]: use `b` instead, centered vertically issue is also auto resolved

**Review by tslmy — APPROVED** [review]

**Pull request merged**